src/substrait/textplan/parser/ParseText.cpp

```diff
@@ -60,11 +61,25 @@ ParseResult parseStream(antlr4::ANTLRInputStream stream) {
         std::current_exception());
   }
 
-  SymbolTable symbolTable = *visitor->getSymbolTable();
+  auto pipelineVisitor = std::make_shared<SubstraitPlanPipelineVisitor>(
+      *visitor->getSymbolTable(), visitor->getErrorListener());
+  try {
+    pipelineVisitor->visitPlan(tree);
+  } catch (...) {
+    parserErrorListener.syntaxError(
+        &parser,
+        nullptr,
+        1,
+        1,
```

Comment:
Can you do something like `/*foo=*/1` so it is more obvious what these arguments are?

Reply:
Done.